docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.pb.bal

```diff
@@ -4,61 +4,60 @@ import ballerina/net.grpc;
 import ballerina/io;
 
 // Non-blocking client
-struct helloWorldStub {
+struct chatStub {
     grpc:Client clientEndpoint;
     grpc:ServiceStub serviceStub;
 }
 
-function <helloWorldStub stub> initStub(grpc:Client clientEndpoint) {
+function <chatStub stub> initStub(grpc:Client clientEndpoint) {
     grpc:ServiceStub navStub = {};
     navStub.initStub(clientEndpoint, "non-blocking", descriptorKey, descriptorMap);
     stub.serviceStub = navStub;
 }
 
-function <helloWorldStub stub> LotsOfGreetings (typedesc listener) returns (grpc:ClientConnection| error)  {
-    var res = stub.serviceStub.streamingExecute("helloWorld/LotsOfGreetings", listener);
-    match res {
-        grpc:ConnectorError err => {
-            error e = {message:err.message};
-            return e;
+function <chatStub stub> chat (typedesc listener) returns (grpc:Client| error)  {
+    var res = stub.serviceStub.streamingExecute("chat/chat", listener);
+        match res {
```

Comment:
`var` and `match` should have the same indentation.

Reply:
Fixed in f2b7cd26cb71721658fbc018e48d1aecaddaecd2